templates/web/base/report/_council_sent_info.html

```diff
@@ -1,6 +1,7 @@
-[% IF problem.whensent || problem.can_display_external_id %]
+[% problem_sent = problem.whensent || c.cobrand.get_body_handler_for_problem(problem).is_council_with_case_management %]
```

Comment:
To keep the template tidier, perhaps `duration_string` could return blank if there's no whensent (after your new bit of code there for case management), and then this template and the other could then print if it wasn't blank (so leaving the logic of whensent + whatever to the code).